pandas/src/inference.pyx

```diff
@@ -538,10 +575,6 @@ def is_time_array(ndarray[object] values):
             return False
     return True
 
-def is_period(object o):
```

Comment:
Yes it is:

- https://github.com/pydata/pandas/blob/master/pandas/src/inference.pyx#L36
- https://github.com/pydata/pandas/blob/master/pandas/src/inference.pyx#L541